### ISSUE TITLE
Display relational algebra expression in the query plan builder

### ISSUE THREAD

**Issue by vogti** [polypheny/Polypheny-DB#344]
Print the query plan as relational algebra expression and allow executing multiple query plans.

**Comment by erdengk** [comment]
Hi, @vogti 
I'm not sure what issue means. Does it mean the same as the following example?

Convert ` select * from S2 where rating>8 `  to `  σ rating>8 (S2) ` 

Perhaps the purpose of the problem is to use relational algebra to generate the lowest cost query plan？

**Comment by vogti** [comment]
Hi @erdengk 
You are right, the description of this issue is not really informative. Polypheny-DB internally represents queries using an extended relational algebra.

The idea of this feature is to come up with a notation for the internal query representation of Polypheny-DB. This would allow to easily test queries independent of a specific query language. However, since we are currently in the process of restructuring and extending this query representation, we should wait with implementing this feature. Furthermore, I expect it to be quite complex due to some recent changes. I think this is rather something for a GSoC project.

**Comment by erdengk** [comment]
@vogti 
Yes, thank you for your answer. I will continue to study this problem.

If you have time, please review this [pr](https://github.com/polypheny/Polypheny-DB/pull/386#issue-1101744016)，thanks.🙏
